src/cli/Cargo.toml

```diff
@@ -66,9 +66,9 @@ content_inspector = { version = "0.2.4", optional = true}
 # Fluvio dependencies
 k8-config = { version = "1.3.0", optional = true}
 k8-client = { version = "5.0.0", optional = true}
-fluvio-cluster = { version = "0.9.1", path = "../cluster", default-features = false, features = ["cli"], optional = true }
+fluvio-cluster = { version = "0.10.0", path = "../cluster", default-features = false, features = ["cli"], optional = true }
 
-fluvio = { version = "0.9.0", path = "../client", default-features = false }
+fluvio = { version = "0.10.0", path = "../client", default-features = false }
```

Comment:
Since we are pre-1.0, we treat the second digit as a "major" version bump, and we use the third digit for "minor" bumps. Can you please change this bump to the crate to `0.9.1`?